<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labIP">
         <property name="text">
          <string>IP :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editIP">
         <property name="placeholderText">
          <string>100.001.101.01</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbConnexion">
         <property name="text">
          <string>connexion</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="labTaille">
         <property name="text">
          <string>Taille : </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QComboBox" name="comboTaille">
           <item>
            <property name="text">
             <string>Petit Cadeau de noël</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Super Cadeau de noël</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Méga Cadeau de noël</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Gestionnaire des livraisons de commandes de noel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QDialogButtonBox" name="BBconfirmation">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="labDescription">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>98</height>
        </size>
       </property>
       <property name="text">
        <string>Veuillez choisir les modalités de livraison</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labNomClient">
         <property name="text">
          <string>Nom du client :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editNomClient">
         <property name="placeholderText">
          <string>ex : Jacques Dutronc</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labJoyeuxNoel">
         <property name="text">
          <string>~ Et Joyeux noël à vous ~</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="labDestination">
           <property name="text">
            <string>Destination :</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QComboBox" name="ComboDestination">
             <item>
              <property name="text">
               <string>France</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Espagne</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Allemagne</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
